artifacts/Restaurants/Restaurants.recipes

```diff
@@ -20,10 +20,10 @@ recipe &displayRestaurants
     list <- list
 
 recipe
-  use as person
+  map 'SYSTEM_user' as user
```

Comment:
Is this the new way to get the system user?

Reply:
It's where I landed, but we can tweak it as needed.